README.md

```diff
@@ -8,8 +8,22 @@ Currently it supports bumping version for a gem.
 ## Bump version
 
 ### Installation
+Dobby require a github app to be installed on your repository or organization wide. If you have already created a github app you can follow instruction from step 3 otherwise follow these steps: 
```

Comment:
```suggestion
Dobby requires a Github App to be installed either on an individual repository or organization wide. If you have already created a Github App you can follow instruction from step 3. Otherwise follow these steps: 
```